perllib/FixMyStreet/App/Controller/Admin.pm

```diff
@@ -1272,7 +1272,9 @@ sub user_edit : Path('user_edit') : Args(1) {
 
         if (!$user->from_body) {
             # Non-staff users aren't allowed any permissions or to be in an area
-            $user->user_body_permissions->delete_all;
+            $user->user_body_permissions->search({
```

Comment:
You could have a User method of admin_user_body_permissions that ignored the trusted one? That would also help with...